tox.ini

```diff
@@ -56,13 +56,12 @@ deps =
 commands =
     flake8 kolibri
 
-##### remove the link checker test because it causes random failures
-#[testenv:docs]
-#changedir=docs/
-#deps =
-#    -r{toxinidir}/requirements/docs.txt
-#commands =
-#    sphinx-build -b linkcheck ./ _build/
+[testenv:docs]
+changedir=docs/
```

Comment:
We were running it, because it was still in the build matrix. But because there was no specific testenv for it, it was running the generic testenv, which is the Python tests!